crates/pet-poetry/src/pyproject_toml.rs

```diff
@@ -38,7 +41,23 @@ fn parse_contents(contents: &str, file: &Path) -> Option<PyProjectToml> {
                     }
                 }
             }
-            name.map(|name| PyProjectToml::new(name, file.into()))
+
+            match name {
+                Some(name) => Some(PyProjectToml::new(name, file.into())),
```

Comment:
Found that the format for pyproject.toml has changed.
Updated to fallback to the new format.

Lets discuss during debt week how we can catch such issues in the future.